tutorials/modules/ROOT/pages/failover-clients-with-hazelcast-cloud.adoc

```diff
@@ -0,0 +1,177 @@
+= Configuring a Failover Client
+:url-code-sample-client-failover: https://github.com/hazelcast/hazelcast-cloud-code-samples/tree/master/client-failover
+:description: In this tutorial, you'll update the code in a Java client to automatically connect to a secondary, failover cluster if it cannot connect its original, primary cluster. You'll also run a simple test to make sure that your configuration is correct and then adjust it to include exception handling.
+:page-layout: tutorial
+:page-product: cloud
+:page-categories: Manage Clusters
+:page-est-time: 30 mins
+
+{description}
+
+== Context
+
+Failover is an important feature of systems that rely on near-constant availability. In Hazelcast, a failover client automatically redirects your connection from one cluster that has problems to a second running cluster.
+
+Consider using a failover client with xref:cloud:ROOT:wan-replication.adoc[WAN replication] as part of your disaster recovery strategy.
+
+[[step-1-setup]]
+== Step 1. Cluster and Client Setup
+
+Create two Serverless clusters that you'll use as your primary and secondary clusters and then download and connect sample Java clients to them.
+
+. xref:cloud:ROOT:create-serverless-cluster.adoc[Create the Serverless cluster] that you'll use as your primary cluster. When the cluster is ready to use, setup details for the Java client are displayed.
+. Follow the instructions in *Java Client Quick Setup* to download, extract, and connect the preconfigured Java client to your primary cluster.
+. xref:cloud:ROOT:create-serverless-cluster.adoc[Create the Serverless cluster] that you'll use as your secondary cluster. 
+. This time when the setup details for the Java client are displayed, from the *Java Client Advanced Setup* section record the following connection details:
+
+** *Cluster name*
+** *Discovery token*
+** *Keystore and truststore password*.
+
+. Repeat step 2 for your secondary cluster. You have two running clusters with connected Java clients.
+. Now that you have checked that both Java clients can connect, shut them down. Either press kbd:[Ctrl+C] or type `exit` at the console. 
+
+[[step-2-configuration]]
+== Step 2. Configure a Failover Client
+
+To create a failover client, update the configuration and code of the Java client for your primary cluster. 
+
+Start by adding the keystore files from the Java client of your secondary cluster.
+
+.	Go to the directory where you extracted the Java client for your secondary cluster and then navigate to `src/main/resources`.
+. Rename the `client.keystore` and `client.trustore` files so they don't overwrite the files in your primary cluster keystore. 
```

Comment:
```suggestion
. Rename the `client.keystore` file to `client2.keystore` and rename the `client.truststore` file to `client2.truststore` to avoid overwriting the files in your primary cluster. 
```

Reply:
Updated as requested.